test/subtype.jl

```diff
@@ -4,6 +4,8 @@ struct Singleton{T<:AbstractVector} <: AbstractSet
     element::T
 end
 space_dimension(s::Singleton) = length(s.element)
+dimension(s::Singleton) = space_dimension(s)
```

Comment:
Wait, I thought this would be 1?
> The affine hull of a singleton (a set made of one single element) is the singleton itself. [[source]](https://en.wikipedia.org/wiki/Affine_hull#Examples)

Reply:
Certainly not equal to `space_dimension` 😄 I was thinking in the LazySet's answer.

I think that `dimension` is conventionally `0` here, [see the part on Vector space dimension here](https://en.wikipedia.org/wiki/Point_(geometry)#Dimension_of_a_point).